lte/gateway/c/oai/tasks/nas/emm/Authentication.c

```diff
@@ -1499,6 +1533,7 @@ static void nas_itti_auth_info_req(
         auth_info_req->resync_param, auts_pP->data,
         sizeof auth_info_req->resync_param);
   }
+  free_wrapper((void**) &ue_context_p);
```

Comment:
I don't understand why you malloced this data structure in the first place.

Reply:
Resolved